Source/g_game.c

```diff
@@ -111,6 +111,7 @@ int             extrakills;    // [crispy] count spawned monsters
 int             totalleveltimes; // [FG] total time for all completed levels
 boolean         demorecording;
 boolean         longtics;             // cph's doom 1.91 longtics hack
+boolean         lowres_turn;          // low-res turning
```

Comment:
Why not just set `longtics` to `false`?

Reply:
`longtics` is only relevant when recording a demo. `shorttics` can be used during normal gameplay if I understand correctly. Also, a variable named `lowres_turn` is used in network PR.

Reply:
It took me forever to understand these variables in prboom+. `longtics` means "store 2 bytes" and `shorttics` means "use lower resolution for turning". So, you can use a lower resolution for turns AND store them as 2 bytes (longtics and shorttics both true).

Reply:
Obviously I want to avoid this mess. I'd prefer there to be only one variable and follow the logic that is outlined here:
https://github.com/chocolate-doom/chocolate-doom/blob/e55356d93a1408ca9db9a8cc57ec243f9fc8f99a/src/doom/d_net.c#L201

Reply:
Choice of demo format may override this logic, i.e. Boom and MBF demos always use shorttics, whereas MB21 demos always use longtics.

Reply:
Maybe we postpone this after the netgame PR merge? It contains some of this code, and I also plan to reorganize the compatibility options a bit, it's too messy now.

Reply:
Okay.